include/quicly/cc.h

```diff
@@ -40,15 +40,10 @@ extern "C" {
 #define QUICLY_MIN_CWND 2
 #define QUICLY_RENO_BETA 0.7
 
-#define SEARCH_DELV_BIN_COUNT 10                 //
-#define SEARCH_SENT_BIN_COUNT (25)               // 10 + 15 extra bins
-#define SEARCH_WINDOW_MULTIPLIER (3.5)
-#define SEARCH_THRESH (0.35)
-
-#define QUICLY_HYBLA_RTT0 (25)
-#define QUICLY_HYBLA_RHO_LIM (16)
-
-// #define SEARCH_EXIT
+#define QUICLY_SEARCH_DELV_BIN_COUNT 10                 // number of search delivered bytes bins
+#define QUICLY_SEARCH_SENT_BIN_COUNT (25)               // number of search sent bytes bins
```

Comment:
Note. in search2.0, we only use delivered bytes for calculation,  could we rename it as QUICLY_SEARCH_BIN_TOTAL.